src/modules/0.0.22/aws_codebuild/entity/build.ts

```diff
@@ -73,4 +73,11 @@ export class CodebuildBuildImport {
     name: 'project_name',
   })
   project: CodebuildProject;
+
```

Comment:
why not always wait? that is the default behaviour for other modules...

Reply:
in the acm request certificate, i made it wait until the cert was valid. I also did the same for codedeploy, and now with codepipeline...

Reply:
that's true, rds also takes a lot of time and we wait for it

Reply:
will change the default behavior!